mithril-aggregator/src/file_uploaders/gcp_uploader.rs

```diff
@@ -480,4 +497,21 @@ mod tests {
             assert_eq!(FileUri(expected_location), location);
         }
     }
+
+    #[tokio::test]
+    async fn retry_policy_from_file_uploader_trait_should_be_implemented() {
+        let expected_policy = FileUploadRetryPolicy {
+            attempts: 10,
+            delay_between_attempts: Duration::from_millis(123),
+        };
+
+        let file_uploader: Box<dyn FileUploader> = Box::new(GcpUploader::new(
+            Arc::new(MockCloudBackendUploader::new()),
+            CloudRemotePath::new("remote_folder"),
+            true,
+            expected_policy.clone(),
+        ));
```

Comment:
The use of `Box` doesn't seem necessary.
```suggestion
        let file_uploader = GcpUploader::new(
            Arc::new(MockCloudBackendUploader::new()),
            CloudRemotePath::new("remote_folder"),
            true,
            expected_policy.clone(),
        );
```